plugin-server/src/worker/vm/lazy.ts

```diff
@@ -26,47 +26,87 @@ export class LazyPluginVM {
     resolveInternalVm!: Promise<PluginConfigVMResponse | null>
     totalInitAttemptsCounter: number
     initRetryTimeout: NodeJS.Timeout | null
+    ready: boolean
+    responseVar: string | null
 
     constructor() {
         this.totalInitAttemptsCounter = 0
         this.initRetryTimeout = null
+        this.ready = false
+        this.responseVar = null
         this.initVm()
     }
 
     public async getExportEvents(): Promise<PluginConfigVMResponse['methods']['exportEvents'] | null> {
-        return (await this.resolveInternalVm)?.methods.exportEvents || null
+        const exportEvents = (await this.resolveInternalVm)?.methods.exportEvents || null
+        if (exportEvents) {
+            await this.setupPluginIfNeeded()
+        }
```

Comment:
To turbo mode this, can we add `&& !this.ready` to all these functions? Keep it in the `setupPluginIfNeeded` code as well for safety.

My reasoning: If we check and return if inside an `awaited` function, we'll most likely consume another tick of the process loop. Since these `get*` functions are called often, exiting early is good.